File: cristopher/Problema0293/problema0293.js
const prompt = require("prompt-sync")({ sigvar: true });

var vetor = []
var somaDosNumeros = 0, media = 0;
var contMaioresMedia = 0, contIguaisMedia = 0, contIguais30 = 0;

for (var i = 0; i < 10; i++) {
    numero = parseFloat(prompt("Digite o número na posição ["+i+"]: "))
    vetor[i] = numero
    somaDosNumeros = somaDosNumeros + vetor[i];
}

media = somaDosNumeros / 10;

for (var i = 0; i < 10; i++) {
    if(vetor[i] > media){
        contMaioresMedia++;
    }

    if(vetor[i] == 30){
        contIguais30++;
    }

    if(vetor[i] == media){
        contIguaisMedia++;
    }
}

console.log("Maiores que a média -> " + contMaioresMedia);
console.log("Iguais a 30 -> " + contIguais30);
console.log("Iguais a média -> " + contIguaisMedia);

File: cristopher/Problema0293/problema0293.py
vetor = []
contIguais30 = 0
contMaioresMedia = 0
contIguaisMedia = 0
somaNumeros = 0

for i in range(0,10):
    numero = float(input(f"Digite o numero na posição [{i}]: "))
    vetor.append(numero)
    somaNumeros = somaNumeros + numero

media = somaNumeros / 10

for i in range(0,10):
    if(vetor[i] > media):
        contMaioresMedia = contMaioresMedia + 1
    if(vetor[i] == 30):
        contIguais30 = contIguais30 + 1
    if(vetor[i] == media):
        contIguaisMedia = contIguaisMedia + 1

print(f"Maiores que a média -> {contMaioresMedia}")
print(f"Iguais a 30 -> {contIguais30}")
print(f"Iguais a média -> {contIguaisMedia}")